app/modules/proposal/controllers/proposal.js

```diff
@@ -0,0 +1,50 @@
+(function() {
+
+    'use strict';
+
+    angular
+        .module('proposal')
+        .controller('proposalController', ['$scope', '$state', 'proposalService', proposalController]);
+
+    function proposalController($scope, $state, proposalService) {
+        $scope.userList = function() {
+            
+            proposalService.getProposalList()
```

Comment:
@gauravc-cuelogic @sandeepp-cuelogic you should fetch proposal list using state provider resolve state.